docs/deploy_prod.sh

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
 set -eu
 
-yarn netlify deploy --site aztec-docs-dev --prod
+yarn netlify deploy --dir /usr/src/docs --site aztec-docs-dev --prod
```

Comment:
@ludamad should we specify `dir` here as well?

Reply:
This should instead be --dir $(pwd) IMO, this is too hard-coded